### PR TITLE
[admin-api] Add AdminAPI utils to template

### PR DESCRIPTION
**Pull request by wlonk**
kitten

### CODE REVIEW

File: tests/test_admin_serializers.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+from sfdo_template_helpers.admin.serializers import AdminAPISerializer
+
+
+class TestAdminAPISerializer:
+    def test_build_url_field(self):
+        class SubclassedAdminAPISerializer(AdminAPISerializer):
+            class Meta:
+                class model:
+                    class _meta:
+                        object_name = "Foo"
```
Comment (cdcarter) [inline tests/test_admin_serializers.py]: 📐

Comment (wlonk) [reply tests/test_admin_serializers.py]: While I appreciate the feedback, I can't say that I understand it.

Comment (cdcarter) [reply tests/test_admin_serializers.py]: oh it was just an amusing amount of single indentation, I'm glad to see this going in :+1:

**Comment by wlonk** [comment]
@davisagli If you'd like to review and, if ✅, merge, I think this is ready.

**Pull request merged**